ecosystem/sep-0006.md

```diff
@@ -814,7 +814,7 @@ Request parameters:
 
 Name | Type | Description
 -----|------|------------
-`id` | string | (optional) The id of the transaction.
+`id` | (optional) string | The id of the transaction.
```

Comment:
I think we should keep this as-is, since it matches the syntax used in our other endpoints and SEPs

Reply:
Reverted the change in https://github.com/stellar/stellar-protocol/pull/972/commits/2672aba7e7d62e3cc830f7b5cc02b6c6326915bf.